queries.sql

```diff
@@ -1,3 +1,34 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals
+postgres-# WHERE name LIKE '%mon';
+
+SELECT name                   
+FROM animals
+WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/01/01';
```

Comment:
- [ ] Kindly change the date after and in this select statement to be `2019/12/31` instead of `2019/01/01` as the query asked to list all animals born between 2016 and 2019 and now if the animal is born for example: `2019/01/02` it will not be returned in the query.

Reply:
Thanks, @gemmen29 for the review. :pray: